chart/values.yaml

```
@@ -0,0 +1,13 @@
+secretName: docker-ecr
+# overrideNamespace:
+
+ecr:
+  region: ""
+  accessKeyId: ""
+  secretAccessKey: ""
```

Regarding my previous question and confidentiality of the secret

```suggestion
  secretName: ""
  awsAccessKeyID_key: ""
  awsSecretAccessKey_key
```